netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -548,7 +584,10 @@ _Requires_exclusive_lock_held_(_net_ebpf_ext_sock_addr_blocked_contexts
 // SOCK_ADDR Program Information NPI Provider.
 //
 
-static const void* _ebpf_sock_addr_specific_helper_functions[] = {(void*)_ebpf_sock_addr_set_redirect_context};
+static const void* _ebpf_sock_addr_specific_helper_functions[] = {
+    (void*)_ebpf_sock_addr_set_redirect_context,
+    (void*)_ebpf_sock_ops_get_current_process_start_key,
```

Comment:
these should all be sock_addr. why are some of these names sock_ops?

everything must be captured on the first invocation of connect/redirect

Reply:
my bad, missed renaming these